pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -215,15 +215,15 @@ private enum LineageUpdateType {
 
   private final String _helixZkURL;
   private final String _helixClusterName;
-  private final String _dataDir;
+  protected final String _dataDir;
   private final boolean _isSingleTenantCluster;
   private final boolean _enableBatchMessageMode;
   private final int _deletedSegmentsRetentionInDays;
   private final boolean _enableTieredSegmentAssignment;
 
   private HelixManager _helixZkManager;
   private HelixAdmin _helixAdmin;
-  private ZkHelixPropertyStore<ZNRecord> _propertyStore;
+  protected ZkHelixPropertyStore<ZNRecord> _propertyStore;
```

Comment:
there is getPropertyStore(), so no need to make this protected.

you added getter for _dataDir, so no need to make it protected either, in fact, _dataDir is from _config which is passed in to the constructor, so you may get it from _config.

Reply:
Will use the getter.